python/pyspark/pandas/correlation.py

```diff
@@ -60,6 +60,7 @@ def compute(sdf: SparkDataFrame, groupKeys: List[str], method: str) -> SparkData
             .alias(CORRELATION_VALUE_2_COLUMN),
         ],
     )
+    spark_session = default_session()
```

Comment:
`sdf` should have a `SparkSession`?

Reply:
Now it's adjusted to "sdf.sql_ctx.sparkSession", thanks!